src/cr/cube/matrix/subtotals.py

```diff
@@ -156,7 +151,7 @@ def _subtotal_row(self, subtotal):
 
 
 class SumSubtotals(_BaseSubtotals):
-    """Subtotal "blocks" created by np.sum() on addends, primarily counts."""
+    """Subtotal "blocks" created by np.sum() on addends and subtrahends, primarily bases."""
```

Comment:
too long. can probably drop the `, primarily bases.` or stretch that to its own sentence in a docstring body. (I would just cut it.)